composer.json

```diff
@@ -17,13 +17,13 @@
     "require": {
         "php":                  "^7.4|^8.0",
         "behat/mink":           "^1.7",
-        "symfony/browser-kit":  "^4.4|^5.0|^6.0",
-        "symfony/dom-crawler":  "^4.4|^5.0|^6.0"
+        "symfony/browser-kit":  "^4.4|^5.0|^6.0|^7.0",
+        "symfony/dom-crawler":  "^4.4|^5.0|^6.0|^7.0"
```

Comment:
Please add CI Process to run Symfony components version 7.0

Reply:
Same here, added 7.0 to the GH actions 👍

Reply:
I'm sorry but CI has not started

Reply:
Its because the test workflow has been disabled due to 60 days of inactivity. You can enable it again in the GH actions tab
<img width="979" alt="Scherm­afbeelding 2024-02-06 om 14 20 26" src="https://github.com/FriendsOfBehat/MinkBrowserKitDriver/assets/4119451/9c152649-03dc-45dd-b066-2b7265cc6ffd">

Reply:
Ready )